Sources/MongoSwiftSync/MongoCollection+Read.swift

```diff
@@ -24,6 +24,30 @@ extension MongoCollection {
         fatalError("unimplemented")
     }
 
+    /**
+     * Finds a single document in this collection that matches the provided filter.
+     *
+     * - Parameters:
+     *   - filter: A `Document` that should match the query
+     *   - options: Optional `FindOneOptions` to use when executing the command
+     *   - session: Optional `ClientSession` to use when executing this command
+     *
+     * - Returns:  the resulting `Document`, or nil if there is no match
+     *
+     * - Throws:
+     *   - `UserError.invalidArgumentError` if the options passed are an invalid combination.
+     *   - `UserError.logicError` if the provided session is inactive.
+     *   - `EncodingError` if an error occurs while encoding the options to BSON.
+     */
+
```

Comment:
extra newline here

Reply:
good catch. this existed in the existing implementation too; fixed both